tests/posix/realpath.c

```diff
@@ -53,6 +59,13 @@ void signal_handler(int sig, siginfo_t *info, void *ctx) {
 	abort();
 }
 
+#define TEST_PATH(rpath, expected) \
```

Comment:
nit: this could be a function, right?

Reply:
I feel like this would make it easier to know which test case failed, since from what I have seen there is no backtrace on a fail.